Bottom: d75feab9097201593b702daf8eded8e985ce5d8b
Top:    451304130741be72defad7390848e57f32e7af60
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-10 19:24:53 +0000

Avoid clearing pins / vias from polygons... it is already done!











---

diff --git a/src/draw.c b/src/draw.c
index 4b02420..dbe197c 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1239,6 +1239,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
       AddPart (Pin);
       return;
     }
+#if 0
   /* Clear the area around the pin */
   if (TEST_FLAG (SQUAREFLAG, Pin))
     {
@@ -1261,6 +1262,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
     {
       gui->fill_circle (Output.pmGC, Pin->X, Pin->Y, half);
     }
+#endif
   if ((!TEST_FLAG (PINFLAG, Pin) && !PCB->ViaOn)
       || (TEST_FLAG (PINFLAG, Pin) && !PCB->PinOn))
     return;
